ynput/ayon-core
# Maya: Fix RedshiftProxy export from renderlayer

## Changelog Description

Do not error on exporting redshift proxy while currently inside a renderlayer.

## Additional info

Note that it does not expose to the artist exporting from a renderlayer. We could add an `EnumDef` on the instance that lists all renderlayers so the artist could potentially pick to export from a specific rendersetup layer if they wanted.

Also fixes that the export would fail if Redshift wasn't loaded.

## Testing notes:

1. Create Render Setup Layer
2. Go into the Render Setup Layer (view it)
3. Export Redshift Proxy should still work (however it will export from the default render layer)
